<?xml version="1.0"?>

<launch>

     <param name="/use_sim_time" value="true"/> 

     <node name="map_server" pkg="map_server" type="map_server" args="$(find multirobot_orca)/map.yaml" >
          <param name="frame_id" value="/map" />
     </node>
-
     <include file="$(find gazebo_ros)/launch/empty_world.launch">
          <arg name="use_sim_time" value="true"/>
          <arg name="debug" value="false"/>
          <arg name="world_name" value="$(find multirobot_fame)/worlds/kaustav1.world"/> 
     </include> 
 
     <!-- <include file="$(find multirobot_orca)/launch/robots_orca.launch" /> 

     <node pkg="collvoid_controller" type="controller.py" name="controller" output="screen"/>
     <node pkg="collvoid_controller" type="collvoid_visualizer.py" name="controller_viz" output="screen"/> -->

     <node pkg="rviz" type="rviz" name="rviz" args="-d $(find multirobot_orca)/launch/rviz_config/orca.rviz" output="screen" /> 

     <!-- <node pkg ="multirobot_orca" name = "robot_model"  type = "robot_model"  output = "screen" args="" /> -->
     <node pkg ="multirobot_orca" name = "robot_model_2"  type = "robot_model_2"  output = "screen" args="" /> 
     <!-- <node pkg ="multirobot_orca" name = "formation_control_dynfeedbacklin"  type = "formation_control_dynfeedbacklin"  output = "screen" args="" /> -->

</launch>
